File: problemas/iniciante/3040/3040.js
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = "";

process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    let lines = input.split('\n');
    let N = parseInt(lines.shift());

    for (let i = 0; i < N; ++i) {
        let [a, d, g] = lines.shift().trim().split(' ').map(Number);

        if ((200 <= a && a <= 300) && d >= 50 && g >= 150) {
            console.log('Sim');
        } else {
            console.log('Nao');
        }
    }
    
});

File: problemas/iniciante/3040/3040.py
def main():
    N = int(input())
    for _ in range(N):
        a, d, g = map(int, input().split())

        if (200 <= a <= 300) and d >= 50 and g >= 150:
            print("Sim")
        else:
            print("Nao")

if __name__ == "__main__":
    main()
